src/components/views/voip/CallView.tsx

```diff
@@ -347,35 +347,38 @@ export default class CallView extends React.Component<IProps, IState> {
     };
 
     private renderCallControls(): JSX.Element {
-        // We don't support call upgrades (yet) so hide the video mute button in voice calls
-        const vidMuteButtonShown = this.props.call.type === CallType.Video;
+        const { call, pipMode } = this.props;
+        const { primaryFeed, callState, micMuted, vidMuted, screensharing, sidebarShown } = this.state;
+
+        // If SDPStreamMetadata isn't supported don't show video mute button in voice calls
+        const vidMuteButtonShown = call.opponentSupportsSDPStreamMetadata() || call.hasLocalUserMediaVideoTrack;
```

Comment:
Any reason we went from `call.type === CallType.Video` to `call.hasLocalUserMediaVideoTrack` ?

Reply:
Call types are a weird thing that don't really work well once you have a call where one side is sending video and the other isn't. Therefore we now have `hasLocalUserMediaVideoTrack` and `hasRemoteUserMediaVideoTrack` which more accurately represent the call's state